src/common/utils/provenance.tsx

```diff
@@ -103,6 +103,53 @@ const metricToSourceMap: RegionSourceMap = {
       url: 'n/a',
     },
   },
+
+  // TODO(8.2)
```

Comment:
Can you add ": Fill in correct sources (or remove if we correctly get them from the API)."

FWIW- I don't 100% remember how provenance stuff works, but hopefully we can get these from the API and don't need frontend to specify them.  We'll need to verify though.